.github/workflows/ports_windows.yml

```diff
@@ -17,7 +17,107 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build:
+  build-vs:
+    strategy:
+      matrix:
+        architecture: [x86, x64]
+        configuration: [Debug, Release]
+        variant: [dev, standard]
+        exclude:
+        - variant: standard
+          configuration: Debug
+    runs-on: windows-2022
```

Comment:
The 2022 probably means VS2022 right? Appveyor was still running VS2017 intentionally to make sure we're somewhat backwards compatible.

Reply:
Yes, VS2022 is what is installed on this image by default. We can update this to install an older version if needed or even add multiple versions of VS to the build matrix.

Reply:
I see that this PR's been updated to test 2017, 2019 and 2022.  That's great!